routes/dao_coin_exchange_with_fees.go

```diff
@@ -908,22 +908,31 @@ const FOCUS_FLOOR_PRICE_DESO_NANOS = 166666
 
 func (fes *APIServer) GetQuoteCurrencyPriceInUsd(
 	quoteCurrencyPublicKey string) (_midmarket string, _bid string, _ask string, _err error) {
-	if IsDesoPkid(quoteCurrencyPublicKey) {
-		// TODO: We're taking the Coinbase price directly here, but ideally we would get it from
-		// a function that abstracts away the exchange we're getting it from. We do this for now
-		// in order to minimize discrepancies with other sources.
-		desoUsdCents := fes.MostRecentCoinbasePriceUSDCents
-		if desoUsdCents == 0 {
-			return "", "", "", fmt.Errorf("GetQuoteCurrencyPriceInUsd: Coinbase DESO price is zero")
-		}
-		price := fmt.Sprintf("%0.9f", float64(desoUsdCents)/100)
-		return price, price, price, nil // TODO: get real bid and ask prices.
-	}
 	utxoView, err := fes.backendServer.GetMempool().GetAugmentedUniversalView()
 	if err != nil {
 		return "", "", "", fmt.Errorf(
 			"GetQuoteCurrencyPriceInUsd: Error fetching mempool view: %v", err)
 	}
+	if IsDesoPkid(quoteCurrencyPublicKey) {
+		usdcProfileEntry := utxoView.GetProfileEntryForUsername([]byte("dusdc_"))
```

Comment:
Oof. Can we please pull this out into a variable somewhere at least? It's referenced below in this function as well I think.

Reply:
made a const called `dusdcProfileUsername` for this.